### PR TITLE
GitHub Icon

### LINKED ISSUE #17: GitHub Icon for cgt
### Background

https://mui.com/material-ui/getting-started/
Mui page has GitHub icon. Want to have the same for the Wordnote
image

here in the red box between the consistency icon and the user icon
image

### TODOs

```
- [ ] Implement GitHub Icon with link https://github.com/ajktown/ConsistencyGPT
```

### TODOs

```
- [ ] TODO: Write your TODOs here
```

### PR DESCRIPTION
**Pull request by akanex115**
# Background
https://github.com/ajktown/ConsistencyGPT/issues/17

## TODOs
- [x] Implement GitHub Icon with link https://github.com/ajktown/ConsistencyGPT

## Checklist (Developers)
- [x] `Draft` is set for this PR
- [x] `Title` is checked
- [x] `Background` is filled
- [x] `TODOs` are filled
- [x] `Assignee` is set
- [x] `Labels` are set
- [x] `Project` is created & linked, if multiple PRs are associated to this PR
- [x] `development` is linked if related issue exists
- [x] `yarn inspect` is run
- [x] `TODOs` are handled and checked
- [x] Final Operation Check is done
- [x] Make this PR as an open PR

## Checklist (Reviewers)
- [x] Check if there are any other missing TODOs that are not yet listed
- [x] Review Code
- [x] Every item on the checklist has been addressed accordingly
- [x] If `development` is associated to this PR, you must check if every TODOs are handled

### CODE REVIEW

File: src/components/atom_appber_parts/index.github_icon.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react'
+import { useOpenNewTab } from '@/hooks/use-open-new-tab'
+import GitHubIcon from '@mui/icons-material/GitHub'
+import StyledIconButtonAtom from '@/atoms/StyledIconButton'
+
+const link = `https://github.com/ajktown/wordnote`
```
Comment (mlajkim) [inline src/components/atom_appber_parts/index.github_icon.tsx]: url different

Comment (mlajkim) [reply src/components/atom_appber_parts/index.github_icon.tsx]: https://github.com/ajktown/ConsistencyGPT

**Review by mlajkim — COMMENTED** [review]
first review

File: src/components/atom_appber_parts/index.github_icon.tsx (hunk 1 above)
Comment (akanex115) [reply src/components/atom_appber_parts/index.github_icon.tsx]: Handled in [dc1b9e3](https://github.com/ajktown/ConsistencyGPT/pull/18/commits/dc1b9e31e27de200b6edd4e192162676dbd6878e)

**Pull request merged**